### PR TITLE
Address misc comments from mounirlamouri@

### PR DESCRIPTION
**Pull request by beaufortfrancois**
Hey @mounirlamouri,

Here's a PR that addresses your feedback raised at #15.
Let me know what you think.

Preview at https://api.csswg.org/bikeshed/?url=https://github.com/beaufortfrancois/picture-in-picture/raw/addressFeedback/index.bs

FIX: #15

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -223,29 +271,13 @@ request could be rejected or even two Picture In Picture windows can be
 created. Regardless, the User Agent will have to fire the appropriate events
 in order to notify the website of the Picture In Picture status changes.
 
+# Security considerations # {#security-considerations}
+
 ## Feature Policy ## {#feature-policy}
 
 This specification defines a <a>feature</a> that controls whether
-{{pictureInPictureEnabled}} is |true| or |false|.
+{{pictureInPictureEnabled}} is <code>true</code> or <code>false</code>.
```
Comment (mounirlamouri) [inline index.bs]: If you go the markdown way, can you do: ``` `true` ``` and ``` `false` ```

File: index.bs [hunk 2]
```diff
@@ -162,48 +182,76 @@ return <a>a new promise</a> |promise| and run the following steps <a>in
...
+2. Return the picture in picture element
+
+## Event types ## {#event-types}
+
+: <dfn event for="HTMLVideoElement"><code>enterpictureinpicture</code></dfn>
+:: Fired on a {{HTMLVideoElement}} when it enters Picture In Picture.
+: <dfn event for="HTMLVideoElement"><code>leavepictureinpicture</code></dfn>
+:: Fired on a {{HTMLVideoElement}} when it leaves Picture In Picture.
+
+# Concepts # {#concepts}
```
Comment (mounirlamouri) [inline index.bs]: I would moke this section on top.

File: index.bs [hunk 3]
```diff
@@ -69,90 +112,67 @@ extensible.
...
-be applied, it SHOULD apply latest ones that were successfully set if any.
-7. Render |video| frames in the |pipVideo|.
-
-Note: It is recommended that the |video| frames are not rendered in the page and
-in the |pipVideo| at the same time but if they are, they must be kept in sync.
+When a video is played in Picture in Picture, the states SHOULD transition as
+if it was played inline. That means that the events SHOULD fire at the same
+time, calling methods SHOULD have the same behaviour, etc. However, the user
+agent MAY transition out of Picture in Picture when the video element enters a
+state that is considered not compatible with Picture in Picture.
```
Comment (mounirlamouri) [inline index.bs]: Maybe this should go to the concepts section?

File: index.bs (hunk 1 above)
Comment (beaufortfrancois) [inline index.bs]: bikeshed doesn't take yet* into account ``true`` sadly ;(

File: index.bs (hunk 2 above)
Comment (beaufortfrancois) [inline index.bs]: Moved.

File: index.bs (hunk 3 above)
Comment (beaufortfrancois) [inline index.bs]: Done.

**Comment by beaufortfrancois** [comment]
@mounirlamouri Feedback addressed.

**Comment by mounirlamouri** [comment]
Argh. I thought you merged this. Sorry :)

**Pull request merged**